### ISSUE TITLE
a question for hacking

### ISSUE THREAD

**Issue by HackerGeo-sp1ne** [danielkrupinski/Osiris#3917]
I've taken a break for many, many months, and now I'd like to get back into the hacking world. I have a main account with lots of skins, lvl 100 and others and I don't know if I'm likely to get my vac ban there? Can it detect me, how to be safe?

**Comment by ofyria** [comment]
No. Even with a paid cheat, unless private/slotted (and even then), don't.
And free cheats when you got skins and stuffs? Can't afford few bucks?

You theorically could though but its risky recently they been banning many providers including top ones (untrusted bans straight) just by modifying the cheat hard enough to pass the sig checks. At least.
But you'll fall on many other problems. Untrusted/vacnet/whatever. If you can't code, no. If you can do a bit, you may use Osiris as a fork and make your own private from scratch with this little base.

So do not. I can suggest you p2c that are selective and slotted, even a 50 slots cheats that never got detected, but, the stuffs you need to get into it are obviously tough. Not only its heavily selective, but, most are invite-based. Aka you have to be either known and active for ages either you buy the invite for hundreds (and risk a ban of your acc cause invites selling aren't allowed anywhere so, if caught, bye money)

Or have friends that can and trust you enough to give thoses.

**Comment by HackerGeo-sp1ne** [comment]
i want to play with others account? generated, not my main account, but I'm afraid I'll get on my main account
thank you

**Comment by ofyria** [comment]
No risks. The only risk you may take is : if your phone number (or mail, probably) or family sharing is on both a cheating account and the main itself. If one falls, the other too.

Well, actually, there is a secondary risk that is linked with the computer itself (hence why some private cheats are having tools to clean your PC before cheating and even HWID spoof) : trustfactor it is. Cheating on one account is a thing, but, any other accounts passing by after that may be impacted and having reduced trust factor and therefore, you'll fall on poor games with plenty of cheaters even without anything enabled/running.

Also, cheats are killing themselves/quitting when you close the game, except for few, but pure dlls like Osiris will. Since they inject into the game and needs the game to function/exist. Exception would be hardware cheats, ring0/kernel cheats for some and so. As well as externals that are running in background and reading memory and not editing it directly if at all. But they COULD remain running, so make sure but if that's a p2c, it'll quit.

Do you need any p2c names? I been cheating since CSGO exists, tried 90% of the providers. Only ones I haven't tried are invites ones (am not that good socially haha) except two. Including novo which is the best atm for legit cheating or even closet/semi-rage. May even do great in rage.

I also know cheap cheats that are not that known and are quite okayish in terms of functionality. Or even quality cheats that are cheap and where you can buy parts (aimbot or visuals, or both for a little more, depending on what you need)

Am not advertising for anyone though. But, Osiris will get you untrusted quite fast. The aimbot is also sluggish, it works but it feels odd, unless you feel like playing with very low settings? You wanna legit closet/semi-rage rage or anything else? I could help even with Osiris but that's at your risks. Also, any free cheats is sigged and detected, doesn't mean you'll get banned straight, they usually wait in order to wave (stacking detected users then hitting them all straight so they grab more, they do it less since the game is free though)

Just tell me what you need, i'll try to help.

**Comment by HackerGeo-sp1ne** [comment]
that's all, thanks a lot for your help, from what I understand I can play without worries for the main account
i will generate an account and i play with osiris legit
